frontend/src/utils/queryGraph.js

```diff
@@ -232,12 +244,27 @@ function prune(q_graph) {
   Object.keys(clonedQueryGraph.nodes).forEach((n) => {
     pruneEmptyArray(clonedQueryGraph.nodes[n], 'category');
     pruneEmptyArray(clonedQueryGraph.nodes[n], 'id');
-    if (clonedQueryGraph.nodes[n].id && clonedQueryGraph.nodes[n].category) {
+    // if (clonedQueryGraph.nodes[n].id && clonedQueryGraph.nodes[n].category) {
+    //   delete clonedQueryGraph.nodes[n].category;
```

Comment:
Can these commented lines be removed?

Reply:
Removed in 74df36f5ada1baa2a9771ec61dfc50831e1aa063